packages/support/lib/image-util.js

```diff
@@ -4,16 +4,10 @@ import { Buffer } from 'buffer';
 import { PNG } from 'pngjs';
 import B from 'bluebird';
 import { hasValue } from './util';
-import log from './logger';
-import { requirePackage } from './node';
-
-
-const OCV_OLD = 'opencv4nodejs';
-const OCV_NEW = '@u4/opencv4nodejs';
+import cv from 'opencv-bindings';
```

Comment:
This module is 8.4 MB in size. Not sure we want to always fetch it no matter what

Reply:
That is why I will put it into its own package that will only ever be required when it will definitely be used (e.g. in the images plugin)